.github/workflows/continuous_integration.yml

```
@@ -35,6 +35,7 @@ jobs:
 
       - name: Making env file
         run: cp .env.example .env
+        run: cp .env.test.example .env.test
```

actionlint (1.7.7)

key "run" is duplicated in element of "steps" section. previously defined at line:37,col:9

(syntax-check)